pkg/rancher-desktop/backend/kube/lima.ts

```diff
@@ -186,7 +187,7 @@ export default class LimaKubernetesBackend extends events.EventEmitter implement
           return k3sConfigString;
         }));
 
-    this.client = new KubeClient();
+    this.client = !process.env.RD_MOCK_FOR_SCREENSHOTS ? new KubeClient() : new KubeClientMock();
```

Comment:
I can't say that I'm very keen on code dedicated to mocking screenshots making it into `lima.ts` and `wsl.ts`. Have we investigated other patterns that we might be able to apply that will achieve something similar?

Reply:
Would we be able to utilize the mock backend that we specify in the factory instead?

https://github.com/rancher-sandbox/rancher-desktop/blob/7e32cb5fa5f99f5a318dbd51c1edb131cb0ae7bd/pkg/rancher-desktop/backend/factory.ts#L15-L17

Reply:
I pushed an update to the PR. I think it's a little bit better than before. Although I don't think it's _perfect_. Would you mind taking a look at it again?